hidapi/hidapi.h

```diff
@@ -221,7 +221,7 @@ extern "C" {
 				-1 on error. If no packet was available to be read within
 				the timeout period, this function returns 0.
 		*/
-		int HID_API_EXPORT HID_API_CALL hid_read_timeout(hid_device *dev, unsigned char *data, size_t length, int milliseconds);
+		int HID_API_EXPORT HID_API_CALL hid_read_timeout(hid_device *device, unsigned char *data, size_t length, int milliseconds);
```

Comment:
In all implementations (in .c files) there is `dev`. I would prefer to change doxygen text to `@param dev` instead of changing `dev` to `device` in function declaration.

Reply:
And rename commit title accordingly

Reply:
On the other hand - in all other places in this `.h` - it is `device`, so either we rename everywhere, or have `device` everywhere (I prefer more readable form for documentation).

Reply:
`hid_device *dev` looks readable to me and `dev` is pretty standard abbreviation for device. But in the end, I would be okay with both options

Reply:
sure
in any case - better if we keep consistency across entire file